bzt/resources/pytest_runner.py

```diff
@@ -74,6 +74,10 @@ def _fill_sample(self, report, call, item, status):
         if call.excinfo is not None:
             self._sample.error_msg = call.excinfo.exconly().strip()
             self._sample.error_trace = "\n".join(traceback.format_tb(call.excinfo.tb)).strip()
+            if call.excinfo.errisinstance(AssertionError):
```

Comment:
@dmand I need your opinion on this change

Reply:
This looks good to me. I've just added a unittest to test_python.py for that.